packages/graphql_server/src/scrapeCategories.ts

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios'
+
+/**
+ * Retrieves the repository topics from GitHub API for the specified repository.
+ * @param repositoryOwner - The owner of the repository.
+ * @param repositoryName - The name of the repository.
+ * @returns A Promise that resolves to a string representing the repository topics.
+ * @throws Error if the repository topics cannot be retrieved.
+ */
+
+async function getRepositoryTopics(
+  repositoryOwner: string,
+  repositoryName: string
+): Promise<string> {
+  const apiUrl = 'https://api.github.com/graphql'
+  const token = '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'
```

Comment:
don't store the key in plain text like that. Instead add a property `GITHUB_API_TOKEN` to the .env and .env.dev files and put the token there. If you want to know how to use process.env variables while testing via node <filename> see my slack post regarding this